src/main/java/net/dv8tion/jda/api/sharding/ShardManager.java

```diff
@@ -1096,28 +1098,53 @@ default VoiceChannel getVoiceChannelById(@Nonnull final String id)
      *
      * @see    net.dv8tion.jda.api.entities.Activity#playing(String)
      * @see    net.dv8tion.jda.api.entities.Activity#streaming(String, String)
+     *
+     * @deprecated
+     *         Use {@link #setActivity(net.dv8tion.jda.api.entities.Activity)} instead
      */
+    @Deprecated
+    @DeprecatedSince("4.0.0")
```

Comment:
The other PR that we just accepted (#1025) uses `@DeprecatedSince("4.BETA.0")`. Do we want to use that here instead?

Reply:
4.BETA.0 is already a released version so saying it was deprecated since then would be a lie. I will probably change it there as well.

Reply:
Sounds good.